src/Sulu/Bundle/MediaBundle/Media/Storage/StorageInterface.php

```diff
@@ -19,45 +19,23 @@
 {
     /**
      * Save the document in a storage and give back the path to the document.
-     *
-     * @param $tempPath
-     * @param $fileName
-     * @param $version
-     *
-     * @return mixed
      */
-    public function save($tempPath, $fileName, $version);
+    public function save(string $tempPath, string $fileName, int $version): string;
 
     /**
-     * Give back the path to the document.
-     *
-     * @param $fileName
-     * @param $version
-     * @param $storageOption
-     *
-     * @return string
+     * Returns the content for the given file as a resource.
      *
-     * @deprecated Deprecated since 1.4, will be removed in 2.0
+     * @return resource
      */
-    public function load($fileName, $version, $storageOption);
+    public function loadAsStream(string $fileName, int $version, string $storageOption);
```

Comment:
only storageOptions should be needed to load the image see https://github.com/sulu/sulu/pull/4024/files#diff-d1603c50279a10b85948f4e515ee33c4